index.bs

```diff
@@ -1160,16 +1146,18 @@ Gets the {{Error}} object passed to {{SensorErrorEventInit}}.
     :: True if sensor instance was associated with a [=platform sensor=],
        false otherwise.
 
-    1.  If |sensor_instance|.{{[[identifyingParameters]]}} is set and
-        |sensor_instance|.{{[[identifyingParameters]]}} allows
-        a unique [=platform sensor=] to be identified, then:
-        1.  let |sensor| be that [=platform sensor=],
-        1.  associate |sensor_instance| with |sensor|.
-        1.  Return true.
-    1.  If the [=sensor type=] of |sensor_instance| has an associated [=default sensor=]
-        and there is a corresponding [=platform sensor=] on the device, then
-        1.  associate |sensor_instance| with [=default sensor=].
+    1.  Let |type| be the [=sensor type=] of |sensor_instance|.
+    1.  If the device has a single [=device sensor=] which can provide [=sensor readings|readings=]
+        for |type|, then
+        1.  Associate |sensor_instance| with a [=platform sensor=] corresponding
+            to this [=device sensor=].
         1.  Return true.
+    1.  If the device has multiple [=device sensors=] which can provide [=sensor readings|readings=]
+        for |type|, then
+        1.  If |type| has an associated [=default sensor=], then
```

Comment:
If platform does not have concept of default sensor, false would be returned, is that expected?

Reply:
but that's not a platform concept, default sensor is defined in an extension API, e.g.  here https://www.w3.org/TR/accelerometer/#model, so if a device has multiple accelerometers, the UA will assume one of them to be the default.

Reply:
thanks, additional note addresses my concern about vague definition of default sensor.

Reply:
Outside this change in Sensor.start(), where you invoke this connect to sensor abstract operation, you should change:

>let connected be the result of invoking connect to sensor.

To

>let connected be the result of invoking connect to sensor with sensor_instance as argument.

Might be a separate patch, or could be rolled into this one since related.